### DISCUSSION TITLE
Custom Cursor is Enabled Only When First App Is Launched

### DISCUSSION THREAD

**Discussion by JustasPolis** [hyprwm/Hyprland#4383] [Q&A] [answered]
Hey does any1 else observe that if I set custom cursor with `exec-once=hyprctl setcursor Bibata-Modern-Ice 24` on hyprland.conf, cursor changes only when I launch some app on startup. Scenario:

1. Reboot
2. Hyprland starts 
3. I observe old cursor
4. I launch kitty or some other app -> I observe new cursor :shrug:.

Shouldn't exec-once set cursor on hyprland startup ?

**Comment by JohRest** [comment]
I've observed the same on my machine...

**Comment by vaxerski** [comment, marked as answer]
dupe of https://github.com/hyprwm/Hyprland/issues/4221
